app/controllers/storage_locations_controller.rb

```diff
@@ -11,7 +11,14 @@ def quantity
   def index
     @selected_item_category = filter_params[:containing]
     @items = current_organization.storage_locations.items_inventoried
-    @storage_locations = current_organization.storage_locations.alphabetized.includes(:inventory_items).class_filter(filter_params)
+
+    @include_inactive_storage_locations = params[:include_inactive_storage_locations].to_i > 0
```

Comment:
Could you confirm the value of `params[:include_inactive_storage_locations]`? I would suspect it to already be a falsey or truthy value?